docs/_snippets/portable-stories-import-annotations.md

```diff
@@ -1,11 +1,32 @@
-```js filename="setupFile.js|ts" renderer="common" language="js"
-// Adjust the import based on the supported framework or Storybook's testing libraries (e.g., react, testing-vue3)
-import { setProjectAnnotations } from '@storybook/your-framework';
+```js filename="setupFile.js|ts" renderer="react" language="ts"
+import { setProjectAnnotations } from '@storybook/react';
+import * as previewAnnotations from './.storybook/preview';
 
-// Replace this export with the default export from your Storybook preview file if you're working with a latest version of Storybook
-import * as projectAnnotations from './.storybook/preview';
+// Apply the global annotations from the Storybook preview file
+const annotations = setProjectAnnotations(previewAnnotations);
+
+// Supports beforeAll hook from Storybook
+beforeAll(annotations.beforeAll);
```

Comment:
@kylegach we can either keep references to beforeAll without adding any imports and just assume people will understand it, or we will have to have specific Jest/Vitest snippets. The tricky part is that this is used in a generic page which doesn't refer to Jest or Vitest specifically. Leaving it as is is fine IMO because jest defines these hooks as globals, so it will just work in Jest, and for Vitest there is a setting for that, and if people copy the code and it breaks, they will know how to fix it